script.js

```diff
@@ -1,4 +1,21 @@
 (function() {
+	const paramString = window.location.search.substr(1);
+	const params = {};
+	const paramArray = paramString.split("&");
```

Comment:
ну и пусть останется только один вид кавычек
```suggestion
	const paramArray = paramString.split('&');
```